### PR TITLE
Add explicitStatus to Tracks and Albums

### PR DESCRIPTION
**Pull request by Tolriq**
Add explicitStatus to Tracks and Albums

### CODE REVIEW

**Review by epoupon — APPROVED** [review]

File: content/en/docs/Responses/AlbumID3.md [hunk 1]
```diff
@@ -127,6 +128,7 @@ description: >
 | `originalReleaseDate` | [`ItemDate`](../itemdate) | No |   **Yes**   | Date the album was originally released. |
 | `releaseDate` | [`ItemDate`](../itemdate)  | No |   **Yes**   | Date the specific edition of the album was released. *Note:* for files using ID3 tags, releaseDate should generally be read from the **TDRL** tag. Servers that use a different source for this field should document the behavior. |
 | `isCompilation` | `boolean` | No |  **Yes**    | True if the album is a compilation. |
+| `explicitStatus` | `string` | No |  **Yes**    | Returns "Explicit" if at least one song is Explicit, "Clean" if no song is Explicit and at least one is "Clean" else "". |
```
Comment (deluan) [inline content/en/docs/Responses/AlbumID3.md:131]: Should we use lowercase for the option: `explicit` or `clean`? Or noting it is case insensitive? IMO TitleCase can lead to unnecessary confusion.

Comment (Tolriq) [reply content/en/docs/Responses/AlbumID3.md:131]: I hesitated with just E and C or int values. I have no strong feelings, but yes if we keep the full values, lowercase is better.

**Review by epoupon — APPROVED** [review]

**Review by deluan — APPROVED** [review]
👍🏼

File: content/en/docs/Responses/AlbumID3.md [hunk 2]
```diff
@@ -127,6 +128,7 @@ description: >
 | `originalReleaseDate` | [`ItemDate`](../itemdate) | No |   **Yes**   | Date the album was originally released. |
 | `releaseDate` | [`ItemDate`](../itemdate)  | No |   **Yes**   | Date the specific edition of the album was released. *Note:* for files using ID3 tags, releaseDate should generally be read from the **TDRL** tag. Servers that use a different source for this field should document the behavior. |
 | `isCompilation` | `boolean` | No |  **Yes**    | True if the album is a compilation. |
+| `explicitStatus` | `string` | No |  **Yes**    | Returns "explicit" if at least one song is explicit, "clean" if no song is explicit and at least one is "clean" else "". |
```
Comment (deluan) [inline content/en/docs/Responses/AlbumID3.md:131]: Needs to list the new field in the OpenSubsonic New Fields section below.

File: content/en/docs/Responses/AlbumID3WithSongs.md [hunk 3]
```diff
@@ -238,6 +241,7 @@ description: >
 | `originalReleaseDate` | [`ItemDate`](../itemdate) | No |   **Yes**   | Date the album was originally released. |
 | `releaseDate` | [`ItemDate`](../itemdate) | No |   **Yes**   | Date the specific edition of the album was released. *Note:* for files using ID3 tags, releaseDate should generally be read from the **TDRL** tag. Servers that use a different source for this field should document the behavior. |
 | `isCompilation` | `boolean` | No |  **Yes**    | True if the album is a compilation.|
+| `explicitStatus` | `string` | No |  **Yes**    | Returns "explicit" if at least one song is explicit, "clean" if no song is explicit and at least one is "clean" else "". |
```
Comment (deluan) [inline content/en/docs/Responses/AlbumID3WithSongs.md:244]: Same

File: content/en/docs/Responses/Child.md [hunk 4]
```diff
@@ -201,6 +202,7 @@ description: >
 | `displayComposer` | `string` | No |  **Yes**   | The single value display composer. |
 | `moods` | Array of `string` | No | **Yes**    | The list of all moods of the song. |
 | `replayGain` | [`ReplayGain`](../replaygain) | No | **Yes**    | The replaygain data of the song. |
+| `explicitStatus` | `string` | No |  **Yes**    | Returns "explicit", "clean" or "". (For songs extracted from tags "ITUNESADVISORY": 1 = explicit, 2 = clean, MP4 "rtng": 1 or 4 = explicit, 2 = clean. See [`albumID3`](../albumid3) for albums) |
```
Comment (deluan) [inline content/en/docs/Responses/Child.md:205]: Same

**Review by deluan — CHANGES_REQUESTED** [review]
Just realized there are some missing copy

File: content/en/docs/Responses/AlbumID3.md (hunk 2 above)
Comment (Tolriq) [reply content/en/docs/Responses/AlbumID3.md:131]: Too long since my last commit :)

**Review by epoupon — APPROVED** [review]

**Review by deluan — APPROVED** [review]

**Review by dweymouth — APPROVED** [review]

**Pull request merged**